src/scripting/lua_ui.cc

```diff
@@ -674,6 +710,116 @@ int LuaPanel::get_child(lua_State* L) {
+
+           * ``"datatype"``: **Mandatory**. The data type of the listselect's entries.
+             Currently only ``"string"`` is supported.
+
+           * ``"entries"``: **Optional**. The entries in the listselect.
+             An array of tables with the following keys:
+
+             * ``"label"``: **Mandatory**. The text for this entry.
+             * ``"value"``: **Mandatory**. The internal value of this entry.
+             * ``"icon"``: **Optional**. The icon filepath for the entry.
+             * ``"tooltip"``: **Optional**. The entry's tooltip.
+             * ``"select"``: **Optional**. Whether to select this entry (default :const:`false`).
+             * ``"indent"``: **Optional**. How many levels to indent the item (default 0).
+
+           * ``"on_selected"``: **Optional**. Callback code to run when the user selects an entry.
+           * ``"on_double_clicked"``: **Optional**.
+             Callback code to run when the user double-clicks on an entry.
+
+         * ``"table"``: A table with multiple rows and columns. Properties:
+
+           * ``"datatype"``: **Mandatory**. The data type of the table's entries.
+             Currently only ``"int"`` is supported.
+           * ``"multiselect"``: **Optional**.
+             Whether the user can select multiple rows (default :const:`false`).
+           * ``"sort_column"``: **Optional**.
+             The index of the column by which the table is sorted (default 0).
+           * ``"sort_descending"``: **Optional**.
+             Whether the sorting order is inverted (default :const:`false`).
+           * ``"columns"``: **Optional**. The columns in the table.
+             An array of tables with the following keys:
+
+             * ``"w"``: **Mandatory**. The width of the column in pixels.
+             * ``"title"``: **Mandatory**. The title of the column.
+             * ``"tooltip"``: **Optional**. The tooltip in pixels.
+             * ``"flexible"``: **Optional**.
+               Whether the column width adapts automatically (default :const:`false`).
+               A table must not have multiple flexible columns.
+             * ``"align"``: **Optional**. The alignment of the column content. Valid values are
+               ``"center"`` (the default), ``"left"``, and ``"right"``.
+
+           * ``"rows"``: **Optional**. The rows in the table.
+             An array of tables with the following keys:
+
+             * ``"value"``: **Mandatory**. The internal value associated with the row.
+             * ``"select"``: **Optional**. Whether to select this row (default :const:`false`).
+             * ``"disable"``: **Optional**. Whether to disable this row (default :const:`false`).
+             * For each column ``i`` (indices are zero-based):
+
+               * ``"text_<i>"``: **Optional**. The text to show in the cell.
+               * ``"icon_<i>"``: **Optional**. The icon filepath in the cell.
```

Comment:
Your example includes a "tooltip_<i\>" option. Did you intentionally left it out from the implementation? I find it useful.

Reply:
That's a mistake in the add-on, Table does not support individual tooltips for table entries.